.github/workflows/squash.yml

```diff
@@ -64,3 +64,30 @@ jobs:
           PV=${{ matrix.python-version }}
           echo "Running tests for Python version  $PV ( ${PV/./} )"
           make test-py"${PV/./}"
+  build-podman-3:
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ['3.6', '3.7', '3.8', '3.9', '3.10', '3.11', '3.12']
```

Comment:
Python 3.6 and 3.7 are EOL, is it necessary to test them?
Would 3.13 be green (rc2 is coming out today)?